.github/ISSUE_TEMPLATE/normal_issue.yaml

```diff
@@ -0,0 +1 @@
+
```

Comment:
Please remove this file too.

Reply:
the file still exists

Reply:
Sorry about that, the file should be removed now.